### DISCUSSION TITLE
Virus detected?

### DISCUSSION THREAD

**Discussion by Yuval1Gutman** [martinet101/ElevenClock#494] [Q&A] [answered]
I just tried to download the latest version of Eleven Clock but my antivirus (Avast Antivirus) says it's detected a virus.
I've downloaded and used Eleven Clock before and it worked great and there wasn't any virus, but from some reason now i can't download it as my antivirus doesn't let me, has this happened to anyone else?

**Comment by marticliment** [comment, marked as answer]
See #489

**Reply by marticliment** [reply]
An updated version has been launched,fixing the virus issue
